airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/task/ProcessRecordsTask.kt

```diff
@@ -9,8 +9,9 @@ import io.airbyte.cdk.message.BatchEnvelope
 import io.airbyte.cdk.message.Deserializer
 import io.airbyte.cdk.message.DestinationMessage
 import io.airbyte.cdk.message.DestinationRecord
-import io.airbyte.cdk.message.DestinationRecordMessage
-import io.airbyte.cdk.message.DestinationStreamComplete
+import io.airbyte.cdk.message.DestinationStreamAffinedMessage
+import io.airbyte.cdk.message.DestinationEndOfStream
+import io.airbyte.cdk.message.DestinationStreamIncomplete
```

Comment:
Where did you define this? You have EndOfStream with streamStatus above?

Reply:
leftover from an early experiment with having separate complete/incomplete messages, but convenient  now that we're switching back :P